beancount_reds_importers/importers/genericpdf/tests/genericpdf_test.py

```diff
@@ -0,0 +1,33 @@
+from os import path
+from beancount.ingest import regression_pytest as regtest
+from beancount_reds_importers.importers import genericpdf
+
+
+@regtest.with_importer(
+    genericpdf.Importer(
+        {
+            "desc": "Paycheck",
+            "main_account": "Income:Salary:FakeCompany",
```

Comment:
Is this correct? Should the `main_account` be the `Income:Salary` account or the `Assets:Bank` account? I know this doesn't matter much but I'm not sure on the philosophy.

Reply:
Yes, I'd prefer to leave it as it is to be consistent, but you're right in that it really doesn't matter.

The only thing this really affects is how you store your Beancount files: if you prefer to use a separate `Income.Salary.FakeCompany.beancount` file as a journal for all your paystub vs. if you prefer to stick these transactions into your `Assets.Bank.beancount` journal. Some users use one large journal file, in which case this makes no difference at all.